### ISSUE TITLE
Not possible to render curly bracket with sub -and superscript

### ISSUE THREAD

**Issue by scaljeri** [mathquill/mathquill#370]
I need to render with MathQuill the following latex:

    \left\{{\text{this}\atop \text{that}}\right=y

You can check it out [here](http://www.codecogs.com/latex/eqneditor.php) to see how it looks like.
In words, its a left curly bracket with sub -and superscript

Is there any way this can be rendered with Mathquill ?

**Comment by laughinghan** [comment]
Hm, I don't think that's currently possible with MathQuill's built-in features, no.

Also, nitpick, I believe "proper" LaTeX should have `\right.` if you don't want a visible right delimiter, like http://mathb.in/26453

**Comment by scaljeri** [comment]
Thanks a lot for the quick reply! Are there any plans to add such a feature ?

**Comment by laughinghan** [comment]
Nothing concrete, unfortunately

**Comment by scaljeri** [comment]
I'm considering doing it myself then and create a PR of course! I must admin, I haven't seen any MathQuill code yet, but do you think adding this feature is almost the same as impossible ?

**Comment by laughinghan** [comment]
No, definitely not impossible! The `dev` branch actually has a pretty decent architecture, I think
